js/treetest.js

```diff
@@ -272,7 +272,7 @@ function bindEmitOnCloseNode() {
 }
 
 function bindEmitOnTaskChanged() {
-	window.addEventListener('taskchanged', ({ detail }) => socket.emitTaskChangedEvent(detail.newTaskIndex));
+	window.addEventListener('taskchanged', ({ detail }) => socket.emitTaskChangedEvent(detail.newTaskIndex + 1));
```

Comment:
Didn't realize that other events are storing task indexes as well, and they are using zero based index. So now there is a discrepancy. 

Don't bother changing other events, lets revert this back to what it was :(